pkg/dataplane/util/ansibleee.go

```diff
@@ -78,12 +80,17 @@ func (a *EEJob) JobForOpenStackAnsibleEE(h *helper.Helper) (*batchv1.Job, error)
 
 	args := a.Args
 
-	playbook := a.Playbook
 	if len(args) == 0 {
-		if len(playbook) == 0 {
-			playbook = CustomPlaybook
+		artifact := a.Role
+		param := "-r"
+		if len(artifact) == 0 {
```

Comment:
Are we still saying role if provided has the highest precedence?

Reply:
oh! I misplaced the order